src/preview.js

```diff
@@ -1,55 +1,41 @@
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
 import countries from '../assets/countries.json';
 import continentNames from '../assets/continent-names.json';
 import continents from '../assets/continents.json';
+import RelatedPosts from './related';
 import { getEmojiFlag } from './utils';
-import { __, sprintf } from '@wordpress/i18n';
 
 export default function Preview( { countryCode, relatedPosts } ) {
-	if ( ! countryCode ) return null;
-
-	const emojiFlag = getEmojiFlag( countryCode ),
-	      hasRelatedPosts = relatedPosts?.length > 0;
+	if ( ! countryCode ) {
+		return <></>;
+	}
+	const emojiFlag = getEmojiFlag( countryCode );
 
 	return (
 		<div className="xwp-country-card">
-			<div className="xwp-country-card__media" data-emoji-flag={ emojiFlag }>
-				<div className="xwp-country-card-flag">
-					{ emojiFlag }
-				</div>
-			</div>
+			<figure
+				className="xwp-country-card__media"
+				data-emoji-flag={ emojiFlag }
+			>
+			</figure>
 			<h3 className="xwp-country-card__heading">
-				{ __( 'Hello from' ) }
-				{ ' ' }
-				<strong>{ countries[countryCode] }</strong>
-				{ ' ' }
-				(<span className="xwp-country-card__country-code">{ countryCode }</span>),
-				{ ' ' }
-				{ continentNames[continents[countryCode]] }!
+				{/* I should use sprintf here, but I am unsure how to do it with markup, for now. */}
+				{ `${__( 'Hello from', 'xwp-country-card' ) } `}
+				<strong>{ countries[ countryCode ] }</strong> (
+				<span className="xwp-country-card__country-code">
+					{ countryCode }
+				</span>
+				), { continentNames[ continents[ countryCode ] ] }!
```

Comment:
Not a full review, but since you've called it out in a comment, here's how it can be done elegantly:

```js
import { createInterpolateElement } from '@wordpress/element';

…

{ createInterpolateElement(
	sprintf(
		// translators: %1$s: is a country name. %2$s: is a country code (e.g. US). %3$s: is a continent name.
		__(
			'Hello from <CountryName>%1$s</CountryName> (<CountryCode>%2$s</CountryCode>), %3$s!',
			'xwp-country-card'
		),
		countries[ countryCode ],
		countryCode,
		continentNames[ continents[ countryCode ] ]
	),
	{
		CountryName: <strong />,
		CountryCode: (
			<abbr
				className="xwp-country-card__country-code"
				title={ countries[ countryCode ] }
			/>
		),
	}
) }
```

Reply:
`createInterpolateElement` - obviously 🤦‍♂️ 
Thanks @delawski !